src/app/containers/hooks/useSubmitMovieFormik.js

```diff
@@ -0,0 +1,20 @@
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+const UseSubmitMovieFormik = (onSubmit) => {
+  return useFormik({
```

Comment:
We should return the jsx directly instead of adding curly braces and return statement 

Ex: const useSubmitMovieFormik = () => useFormik()

Reply:
hook was deleted but point noted :D